Makefile

```diff
@@ -46,13 +47,16 @@ $(LOCAL_BRIDGE_PLUGIN_BINARY): $(SOURCES)
 generate: $(SOURCES)
 	go generate -x ./pkg/... ./plugins/...
 
-.PHONY: unit-test integration-test
+.PHONY: unit-test integration-test e2e-test
 unit-test: $(SOURCES)
 	go test -v -cover -race -timeout 10s ./pkg/... ./plugins/...
 
 integration-test: $(SOURCE)
 	go test -v -tags integration -race -timeout 10s ./pkg/... ./plugins/...
 
+e2e-test:  $(SOURCE) plugins
+	sudo -E CNI_PATH=${ROOT}/bin/plugins ${GO_EXECUTABLE} test -v -tags e2e -race -timeout 60s ./plugins/...
```

Comment:
Since we rely on PATH for finding the go executable anyway, what's the advantage to `${GO_EXECUTABLE}` here instead of simply `go`?